### PR TITLE
ADMIN can associates reviewer to technology manualy

### LINKED ISSUE #672: Permitir atribuir manualmente um curador para uma tecnologia
## Feature Description

Hoje a distribuição dos curadores dos cadastros de tecnologias ocorre de forma automatizada. Para que não ocorra gargalos no processo é preciso permitir que o ADMIN possa fazer a designação do curador de forma manual também.
- Essa funcionalidade pode ficar no item TECNOLOGIAS no ambiente ADMIN.
- Pode designar qualquer usuário com o papel de REVIEWER
---------------

## Acceptance criteria

* 

## Implementation Brief

*

### PR DESCRIPTION
**Pull request by alexandreadames**
## Summary

Resolves #672 

### Rota

| Description | Method | Route | Body | Auth |
| --- | --- | --- | --- | --- |
|ADMIN can associates reviewer to technology manualy. |PUT| /technologies/:id/reviewer | reviewer (reviewer id) | Yes:ADMIN | 
Se a tecnologia já tiver um curador ele é excluido e notificado. O novo curador é associado e também é notificado.

## QA Steps

Rodar: `technology.spec.js``

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

File: packages/api/app/Validators/TechnologyReviewer.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class TechnologyReviewer {
```
Comment (mateus4k) [inline packages/api/app/Validators/TechnologyReviewer.js:1]:
```suggestion
class UpdateTechnologyReviewer {
```

File: packages/api/resources/views/emails/technology-revision-revoked.edge [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<h2> Olá {{ userOldReviewer.full_name }} </h2>
+
+<p>
+  Sua revisão paa a tecnologia {{technology.title}} foi revogada na Plataforma Sabiá. <br />
```
Comment (mateus4k) [inline packages/api/resources/views/emails/technology-revision-revoked.edge:4]:
```suggestion
  Sua revisão para a tecnologia {{technology.title}} foi revogada na Plataforma Sabiá. <br />
```

**Review by mateus4k — CHANGES_REQUESTED** [review]
Nice!

**Review by mateus4k — APPROVED** [review]

**Review by NaylsonFerreira — APPROVED** [review]
Top

**Pull request merged**